From 6ca526df40590ce9d52f42c68c58826b1a361bf0 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Mon, 17 Jun 2019 13:51:55 -0700
Subject: [PATCH] OS-7841 remove datasets.joyent.com usage from imgadm test
 suite (missing update to manifest) Reviewed by: Todd Whiteman
 <todd.whiteman@joyent.com> Approved by: Todd Whiteman
 <todd.whiteman@joyent.com>

---
 src/manifest | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/manifest b/src/manifest
index 97e0ea00..7c096579 100644
--- a/src/manifest
+++ b/src/manifest
@@ -427,7 +427,6 @@ f usr/img/test/IMG.test.js 0444 root bin
 f usr/img/test/basics.test.js 0444 root bin
 f usr/img/test/create.test.js 0444 root bin
 f usr/img/test/docker.test.js 0444 root bin
-f usr/img/test/dsapi.test.js 0444 root bin
 f usr/img/test/import.test.js 0444 root bin
 f usr/img/test/mk-custom-image 0555 root bin
 f usr/img/test/runtests 0555 root bin
-- 
2.21.0

